app/src/helpers/index.ts

```diff
@@ -1,26 +1,28 @@
-export const getChainConfig = (chain: string) => {
-  switch (chain) {
-    case 'mainnet':
-      return {
-        endpoint: 'https://api.etherscan.io',
-        apiKey: process.env.ETHERSCAN_API_KEY,
-      }
-    case 'goerli':
-      return {
-        endpoint: 'https://api-goerli.etherscan.io',
-        apiKey: process.env.ETHERSCAN_API_KEY,
-      }
-    case 'sepolia':
-      return {
-        endpoint: 'https://api-sepolia.etherscan.io',
-        apiKey: process.env.ETHERSCAN_API_KEY,
-      }
-    case 'optimism':
-      return {
-        endpoint: 'https://api-optimistic.etherscan.io',
-        apiKey: process.env.ETHERSCAN_API_KEY_OPTIMISM,
-      }
-    default:
-      throw new Error('Not supported chainID')
-  }
+import type { SupportedChain } from '~/types'
+
+export const chainConfig: Readonly<
+  Record<SupportedChain, { endpoint: string; apiKey: string | undefined }>
+> = {
+  mainnet: {
+    endpoint: 'https://api.etherscan.io',
+    apiKey: process.env.ETHERSCAN_API_KEY,
```

Comment:
Would this mean that we will expose secrets to frontend? Or are they simply undefined on the frontend?

Reply:
it will be undefined